service/src/main/kotlin/fi/espoo/evaka/daycare/controllers/UnitAclController.kt

```diff
@@ -65,6 +65,34 @@ class UnitAclController(private val acl: AccessControlList) {
         return ResponseEntity.noContent().build()
     }
 
+    @PutMapping("/daycares/{daycareId}/specialeducationteacher/{employeeId}")
```

Comment:
I wonder whether we should do this in a more generic way (since it's pretty much the same as with `/daycares/{daycareId}/supervisors/{employeeId}` mapping. We could have one generic endpoint for both, supervisor and specialeducationteacher. Even required roles are the same. But then again, I wonder if it's worth it since there are only two cases.